DirectEmployers/MyJobs
# Move error handling from filter to process_response because apache do…

## What's this PR do?
Moves error handling from a filter to process_response, since we ran into so many problems with the filter yesterday.

## Where should the reviewer start?
The filter has been moved to the middleware in monitor.py.
This required some of the logic of when the middleware needs to be called, which is why it was moved in default_settings.py

## How should this be manually tested?
Set `DATADOG_TRACE['ENABLED'] = True` in settings.py
Make sure you have aws credentials in secrets.
Cause a few errors.
Check the logstash-error-* index in Kibana (ask me how if you don't know how to). Your error should be there. 

## Any background context you want to provide?
Because of the issues we ran into yesterday, I'd still like to load test this in staging before deploying to production to confirm that it is going to be an ok option.


## PR Checklist

### Separation of Concerns
